lib/cadet_web/router.ex

```diff
@@ -28,6 +28,21 @@ defmodule CadetWeb.Router do
   # Authenticated Pages
```

Comment:
We need a path to get user information, such as username and (later on) progress on the story. Perhaps a `/user` path?

Reply:
I think we will probably need a full `users` controller. I can see the need to implement more features as we go along (notification preferences, display picture, whatnot)